src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -289,6 +369,26 @@
                 event.preventDefault();
                 this.loadAction({projectId, versionId});
             },
+            handleEditVersionNote(projectId: number, versionId: string, versionNumber: number) {
+                this.projects.filter(project => {
+                    if (project.id === projectId) {
+                        this.displayProjectName = project.name
+                        this.editedNote = project.versions
+                            .find(version => version.versionNumber === versionNumber)!.note || ""
+                    }
+                })
+                this.versionNoteToEdit = {projectId, versionId}
+                this.selectedVersionNumber = `v${versionNumber}`;
+            },
+            handleEditProjectNote(projectId: number) {
+                this.projects.find(project => {
+                    if (project.id === projectId) {
+                        this.displayProjectName = project.name
+                        this.editedNote = project.note || ""
+                    }
+                })
+                this.projectNoteToEdit = projectId
```

Comment:
```suggestion
                const project = this.projects.find(p => p.id === project.id);
                this.displayProjectName = project.name;
                this.editedNote = project.note || "";
                this.projectNoteToEdit = projectId;
```

A little more readable?